website/client/tests/unit/libs/highlightUsers.spec.js

```diff
@@ -17,6 +17,24 @@ describe('highlightUserAndEmail', () => {
     expect(result).to.contain('<span class="at-text at-highlight">@user</span>');
   });
 
+  it('highlights username sandwiched with underscores', () => {
+    const text = 'hello @_user_';
+
+    const result = highlightUsers(text, '_user_', 'displayedUser');
+    expect(result).to.contain('<span class="at-text at-highlight">@_user_</span>');
+    expect(result).to.not.contain('<em>');
+    expect(result).to.not.contain('</em>');
+  });
+
+  it('highlights username sandwiched with double underscores', () => {
+    const text = 'hello @__user__';
```

Comment:
Similar to the test case above, the html tags need to already be present in the input data to provide a realistic scenario.

```suggestion
    const text = 'hello @<strong>user</strong>';
```